core/src/cubos/core/io/glfw_window.hpp

```diff
@@ -37,9 +37,15 @@ namespace cubos::core::io
         void setCursor(std::shared_ptr<Cursor> cursor) override;
         void setClipboard(const std::string& text) override;
         const char* getClipboard() const override;
+        bool keyPressed(Key key) const override;
+        bool keyPressed(Key key, Modifiers modifiers) const override;
+
+        Modifiers modifiers() const;
```

Comment:
Why not make this virtual and exposed on `BaseWindow`?

Reply:
Querying modifiers is misleading, eg. once you press "Control" the modifiers mask will be still be 0, only when you press the next key it will be Modifiers::Control.

I think it's better not to expose it. GLFW also doesn't expose it, probably for the same reasons.